tools/cibsecret.in

```diff
@@ -118,15 +135,15 @@ check_usage() {
         unstash)    [ "$2" -ne 3 ] && usage 1 ;;
         delete)     [ "$2" -ne 3 ] && usage 1 ;;
         sync)       [ "$2" -ne 1 ] && usage 1 ;;
-        --help)     usage 0 ;;
+        --help)     usage $CRM_EX_OK ;;
         --version)  crm_attribute --version; exit $? ;;
-        *)          usage 1 ;;
+        *)          usage $CRM_EX_USAGE ;;
     esac
 }
 
 fatal() {
     echo "ERROR: $*"
-    exit 1
+    exit $CRM_EX_ERROR
```

Comment:
While this would be sufficient, this is a good opportunity to give more detailed status than it used to. Maybe fatal's first argument could be an exit status.

Also, in the normal case, the script just exits with whatever the exit status of cibsecret_$cmd is. Sometimes that's OK but sometimes it's the result of system commands like ssh, which may or may not match our standard codes.

Reply:
Done. Tell me if you got any better suggestions for any of the returns (it's hard to find ones that fit 100% to the situation and for some there might be 2 codes that makes the same amount of sense).